src/pedestal_toolbox/params.clj

```diff
@@ -68,6 +68,21 @@
             (assoc ctx :response (response/bad-request e)))))})
     {:schema schema}))
 
+(def query-param-content-type
+  "An enter interceptor that fakes an Accept header so that later
+  interceptors can handle the Accept header normally. This is used
+  because it's literally impossible to make a clickable link in a
+  browser that sets the Accept header in the normal way.
+
+  To use, add an content-type=application/csv to the URL query string.
+  Expects the query params to have already been kewordized by kewordize-params"
+  (interceptor
+   {:enter
+    (fn [ctx]
+      (if-let [content-type (get-in ctx [:request :params :content-type])]
+        (assoc-in ctx [:request :headers "accept"] content-type)
+        ctx))}))
```

Comment:
I don't like how we're confusing content-type and accept in this code. "content-type" is for specifying the encoding of data you're sending to the other side (server to client or client to server), "accept" is for requesting the format you'd like your response encoded in.

Can we change the query param to "accept"?